src/main/java/com/example/enterprise/service/ExpenseService.java

```diff
@@ -28,7 +28,7 @@ public void save(Expense expense) {
     }
 
     @Override
-    public List<Expense> showAll() {
+    public List<Expense> showAllExpenses() {
```

Comment:
Changed the method name from showAll() to showAllExpenses() because showAll doesn't specify what to show. An outsider would not understand what it is trying to show. showAllExpenses() name is much more meaningful and provides more clarity to the code logic.

Reply:
Since the method is in "ExpenseService", expense is a given.  So, showAll() is acceptable in this state, and showAllExpenses() could also be acceptable.